### PR TITLE
MB-9535 fix billable weight hint texts

### PR DESCRIPTION
**Pull request by YanZ777**
## Description

Fixed some of the logic around the hint text, refactored some tests, took out the console output being generated from the tests.

## Reviewer Notes

Is the logic around the hint text correct?
Should there be other tests? 
Are the tests split out ok?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make office_client_run
make server_run
```

The easiest way might be to change the move with the locator `MAXCED`. 
I adjusted the max billable weight on that to be 1500 and the shipment weight to be 1560, and then played with those values.

For hint text that shouldn't be there on the edit max billable weight page, you can use the move with the locator `NOESTW`

## Code Review Verification Steps
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-9535) for this change

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

image

### CODE REVIEW

**Review by Shkeating — APPROVED** [review]

File: src/components/Office/BillableWeight/EditBillableWeight/EditBillableWeight.jsx [hunk 1]
```diff
@@ -17,20 +17,27 @@ function BillableWeightHintText({
   originalWeight,
   totalBillableWeight,
 }) {
-  const showToFit = billableWeight > maxBillableWeight && billableWeight < estimatedWeight * 1.1;
+  const estimatedWeightTimes110 = estimatedWeight * 1.1;
+
+  const showToFit = billableWeight > maxBillableWeight && billableWeight > estimatedWeightTimes110;
```
Comment (donaldthai) [inline src/components/Office/BillableWeight/EditBillableWeight/EditBillableWeight.jsx:22]: So looking at the display logic, shouldn't this show when the `max billable weight` is exceeded by the `total billable weight` AND the `toFitValue` is below the 110% estimated weight?
image

**Review by donaldthai — COMMENTED** [review]
I'm seeing a weird issue where the default value of the input on the edit billable weight form is using the max billable weight value: 
Oct-04-2021 16-18-31

**Comment by YanZ777** [comment]
> I'm seeing a weird issue where the default value of the input on the edit billable weight form is using the max billable weight 
Good catch! I've fixed this.

File: src/components/Office/BillableWeight/EditBillableWeight/EditBillableWeight.jsx (hunk 1 above)
Comment (YanZ777) [reply src/components/Office/BillableWeight/EditBillableWeight/EditBillableWeight.jsx:22]: Hmmmmm that doesn't sound quite right either, but I do think my signs are flipped here. I'll wait for Claire to weigh in on this.

**Review by cosburn — APPROVED** [review]
Looks like the logic is working as expected!

**Review by donaldthai — APPROVED** [review]
Code looks good! Since Claire is happy with the functionality, I'm happy!

**Pull request merged**